.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseNumberParser.cs

```diff
@@ -328,6 +336,9 @@ protected ParseResult TextNumberParse(ExtractResult extResult)
                 }
 
                 result.Value = intPartRet + pointPartRet;
+
+                // Get RelativeTo for Ordinal

```

Comment:
Nit: Change the comment to something like: "// Every ordinal number is relative to the start."